docs/current_docs/api/clients-sdk.mdx

```diff
@@ -158,6 +164,34 @@ In this structure:
 - `src/` is where your Dagger module code goes. It contains sample code to help you get started.
 - `sdk/` contains the PHP SDK.
 
+</TabItem>
+<TabItem value="Java">
+```
+.
+├── LICENSE
+├── dagger.json
+├── pom.xml
+├── src
+│   └── main
+│       └── java
+│           └── io
+│               └── dagger
+│                   └── modules
+│                       └── mymodule
+│                           ├── MyModule.java
+│                           └── package-info.java
+└── target
+
+9 directories, 5 files
+```
+
+In this structure:
+
+- `dagger.json` is the [Dagger module configuration file](../configuration/modules.mdx).
+- `pom.xml` manages the module dependencies.
+- `src/main/java/io/dagger/modules/mymodule/` is where your Dagger module code goes. It contains sample code to help you get started.
+- `target` contains the compiled Java classes.
```

Comment:
```suggestion
- `target` contains the generated Java source classes.
```

In this specific case, we are only using `target` to store the result of the codegen so that it helps IDEs for code completion. But there will not have compiled classes generated, it's all source code.